src/webview/bundle/style.scss

```diff
@@ -0,0 +1,41 @@
+.buttonStyle {
+    text-align: center;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    color: var(--vscode-button-foreground);
+  
+    &:hover {
+        background-color: '#BE0000' !important;
+    }
+  
+    text-transform: none;
+}
+
+.mainContainer,
+.formContainer,
+.form {
+    display: flex;
+    flex-direction: column;
+    font-family: var(--vscode-font-family);
+    // overflow-y: scroll;
+}
+
+.margin {
+    margin: 3rem;
+}
+
+.title {
+    width: 100%;
+    margin-bottom: 1rem;
+}
+
+.subTitle {
+    margin-bottom: 2rem;
+    max-width: 59rem;
+    justify-content: center;
+    word-spacing: 5px;
```

Comment:
Either use `rem` or `px`. Do not use both.

Reply:
I am using percentage now